config/cesm/machines/config_machines.xml

```diff
@@ -559,76 +559,76 @@ This allows using a different mpirun command to launch unit tests
         <command name="load">mkl</command>
       </modules>
       <modules compiler="gnu">
-        <command name="load">gnu/9.1.0</command>
-        <command name="load">openblas/0.3.6</command>
+        <command name="load">gnu/10.1.0</command>
```

Comment:
Have you confirmed that this doesn't change answers for any of our CESM testing? It would be good to know that before merging.

Reply:
In the past, I think we've said we should do at least prealpha testing when doing a major compiler version update like this. @fischer-ncar , if you or @jedwards4b hasn't already done so, would you be able to run the cheyenne_gnu prealpha tests with baseline comparisons? (And prebeta cheyenne_gnu if you feel it's warranted; I notice that prealpha cheyenne_gnu has very little coverage.)

Reply:
@billsacks I've start prealpha and prebeta tests for cheyenne_gnu.

Reply:
Testing is finished.  gnu/10.1.0 is bfb with gnu/9.1.0.

Reply:
Awesome, thanks!

Reply:
Just FYI, we did find a case where answers changed because of this update. See https://github.com/ESCOMP/CTSM/issues/1464

I want to highlight it just to reinforce that our paranoia over compiler updates is often justified. :-) It did take us a bit to track it down to this change. It's also helpful that the compiler change was in one PR and not mixed with anything else, so thanks for that.